integration-tests/src/test/scala/tech/beshu/ror/integration/utils/ESVersionSupport.scala

```diff
@@ -34,8 +34,7 @@ trait ESVersionSupportForAnyWordSpecLike extends ESVersionSupport {
     string.taggedAs(firstTestTag, otherTestTags: _*)
 }
 
-trait ESVersionSupportForAnyFreeSpecLike extends ESVersionSupport {
-  this: AnyFreeSpecLike =>
+trait ESVersionSupportForAnyFreeSpecLike extends ESVersionSupport with AnyFreeSpecLike {
```

Comment:
why do we have to do it like that?

Reply:
Scala 3 compiler does not allow access to protected class FreeSpecStringWrapper otherwise. It may or may not have something to do with https://docs.scala-lang.org/scala3/reference/dropped-features/this-qualifier.html (it is not the same, but perhaps some side effect of more strict checks)